dataactcore/migrations/versions/812387580a0b_rename_user_permissions_column.py

```diff
@@ -0,0 +1,44 @@
+"""rename user permissions column
+
+Revision ID: 812387580a0b
+Revises: a97dabbd44f4
+Create Date: 2016-11-09 11:40:11.657516
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '812387580a0b'
+down_revision = 'a97dabbd44f4'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_data_broker():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('users', sa.Column('permission_type_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'users', 'permission_type', ['permission_type_id'], ['permission_type_id'])
+    op.drop_column('users', 'permissions')
+    ### end Alembic commands ###
+
+
+def downgrade_data_broker():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('users', sa.Column('permissions', sa.INTEGER(), autoincrement=False, nullable=True))
+    op.drop_constraint(None, 'users', type_='foreignkey')
```

Comment:
At line 41, the user table has both columns, so that's where we'd migrate the data. As you say, we wouldn't want to use the `User` model directly (though we could replicate it) as it wouldn't have the right fields. Here's one example: https://github.com/fedspendingtransparency/data-act-broker-backend/blob/development/dataactcore/migrations/versions/a97dabbd44f4_drop_filetype_validation_table.py#L51 

In any event, I think there are more important issues at hand. As long as we're okay with the remote possibility of permission-less users, let's not worry about it.